velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -105,6 +105,99 @@ class PrestoSerializerTest : public ::testing::Test {
     return vectorMaker_->rowVector(childVectors);
   }
 
+  void writeInt32(OutputStream* out, int32_t value) {
+    out->write(reinterpret_cast<char*>(&value), sizeof(value));
+  }
+
+  void writeInt64(OutputStream* out, int64_t value) {
+    out->write(reinterpret_cast<char*>(&value), sizeof(value));
+  }
+
+  // Take a base vector and serialize it as an RLE vector
```

Comment:
Comments should be full sentences, i.e. start with a capital letter and end with a period. Please, fix throughout.

Reply:
done